src/mixins/filing-mixin.ts

```diff
@@ -140,7 +140,7 @@ export default class FilingMixin extends Mixins(DateMixin) {
       },
       correction: {
         correctedFilingId: correctedFiling.header.filingId,
-        correctedFilingType: FilingTypes.REGISTRATION,
+        correctedFilingType: correctedFiling.header.name,
```

Comment:
This might be `registration` or `changeOfRegistration`.

This is OK to be hard-coded on line 112 because atm only BEN IA corrections are supported, but when we make all BEN corrections go through this flow then that will need to be updated as well.

Reply:
Hopefully we'll remember when we rename that method from buildIaCorrectionFiling() to buildBenCorrectionFiling() and also change the code that calls it! 😁